src/main/scala/org/phenoscape/kb/CharacterDescription.scala

```diff
@@ -198,8 +198,9 @@ case class CharacterDescription(iri: IRI, description: String, matrix: Character
     "@id" -> iri.toString.toJson,
     "description" -> description.toJson,
     "matrix" -> matrix.toJSON,
-    "character id" -> character.iri.toString.toJson,
-    "character label" -> character.label.toJson).toJson.asJsObject
+    "character" -> Map("@id" -> character.iri.toString.toJson,
+      "label" -> character.label.toJson).toJson.asJsObject
```

Comment:
Could you use `"character" -> character.toJSON` instead of creating the Map here?